kie-server-parent/kie-server-services/kie-server-services-kafka/src/test/java/org/kie/server/services/jbpm/kafka/KafkaEventWriterReaderTest.java

```diff
@@ -58,13 +59,17 @@ public void testEventWriter() {
     public void testEventReader() {
         final String topic1 = "test1";
         final String topic2 = "test2";
+        final String topic3 = "test3";
         final String property1 = KafkaServerUtils.TOPIC_PREFIX + topic1 + "." + "eventReaderClass";
         final String property2 = KafkaServerUtils.TOPIC_PREFIX + topic2 + "." + "eventReaderClass";
+        final String property3 = KafkaServerUtils.TOPIC_PREFIX + topic3 + "." + "eventReaderClass";
```

Comment:
What I'm missing here is a test to make sure we meet the requirements for this ticket, i.e. instead of setting a reader/writer for a specific topic, I would also make sure to test that a reader/writer is set properly for all topics by using below system property:

- `org.kie.server.jbpm-kafka.ext.eventReaderClass` 
- `org.kie.server.jbpm-kafka.ext.eventWriterClass`